.gitlab-ci.yml

```diff
@@ -0,0 +1,5 @@
+include:
```

Comment:
This file would need to be removed

Reply:
Yes, it's not needed here. I'm sorry my fault.